libs/wingsdk/src/cloud/bucket.ts

```diff
@@ -457,6 +469,21 @@ export interface IBucketClient {
    * @inflight
    */
   rename(srcKey: string, dstKey: string): Promise<void>;
+
+  /**
+   * Initiate a multipart upload to a given key in the bucket.
+   * @param key The key to upload to
+   * @returns The upload id for the multipart upload
+   * @inflight
+   */
+  multipartUpload(key: string): Promise<string>;
```

Comment:
Where do I specify the content?

Reply:
this just initiates an upload, you the either use a signed url and an http put or the `uploadPart` method to specify the content.